examples/training/AutoMLOps/components/component_base/src/Custom train model.py

```diff
@@ -0,0 +1,148 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
should this filename be lowercase w underscores for consistency?

Reply:
Good catch - this was actually a bug that I hadn't seen before, had to do with the mixing KFP and non-KFP defined components. I looked into it and actually found that we don't need the user to specify the `use_kfp_spec` parameter; this is something we can determine at runtime by checking the temporary yaml file.